### ISSUE TITLE
Cannot build ortools from binary with jdk8

### ISSUE THREAD

**Issue by unfuse** [google/or-tools#1191]
I attempted to use OR-Tools installed from binary (for Java, on Mac) in a local project, and encountered an error in compilation. Error below: 

>bad class file: /Users/david.kennedy/Downloads/or-tools_MacOsX-10.14.2_v7.0.6546/lib/com.google.ortools.jar(com/google/ortools/graph/LinearSumAssignment.class)
>    class file has wrong version 54.0, should be 52.0
>    Please remove or make sure it appears in the correct subdirectory of the classpath.`

The [instructions](https://developers.google.com/optimization/install/java/mac) state you need jdk 7, and i have jdk 8, but it seems like the jar was built with jdk 10 

>cat MANIFEST.MF 
>Manifest-Version: 1.0
>Created-By: 10.0.1 (Oracle Corporation)

**Comment by lperron** [comment]
We should be building using jdk8.
If not, it is a bug. We will check.

Thanks
Laurent Perron | Operations Research | lperron@google.com | (33) 1 42 68 53
00



Le ven. 12 avr. 2019 à 17:06, David Kennedy <notifications@github.com> a
écrit :

**Comment by Mizux** [comment]
When performing a brew update, I got Java 10 (which is the current default)

On my way to [setup java8](https://stackoverflow.com/questions/24342886/how-to-install-java-8-on-mac).

**Comment by unfuse** [comment]
Thanks much! I noticed that the download link from the install site doesn't work anymore (https://github.com/google/or-tools/releases/download/v7.0/or-tools_MacOsX-10.14.2_v7.0.6546.tar.gz) - says not found. It looks like ...MacOsX-10.14.2... should be ...MacOsX-10.14.4...?

However, I tested by downloading it directly from https://github.com/google/or-tools/releases and built it with my local project and it works!
